stonesoup/models/measurement/tests/test_models.py

```diff
@@ -373,7 +273,7 @@ def h3d_rr(state_vector, pos_map, vel_map, translation_offset, rotation_offset,
         (   # 3D meas, 6D state
             h2d_rr,  # h
             CartesianToBearingRangeRate,  # ModelClass
-            np.array([[200], [10], [0], [0], [0], [0]]),  # state_vec
+            np.array([[200.], [10.], [0.], [0.], [0.], [0.]]),  # state_vec
```

Comment:
If this is `# state_vec` is there a reason not to use a `StateVector`

Reply:
No, just user error